sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
> Why would you set minBE = maxBE = 0 for CCA? In my understanding just checks if the medium is currently free for sending. If this is not the case, the transceiver initiates another channel sensing according to the configured backoff time. So why would you fix this backoff time if CCA is enabled and why would you disable retransmissions in this case?

To emulate what kw2xrf is doing. `minBE = maxBE = 0` effectively turns off (random) backoff, so that the frame is immediately sent if the channel is clear. Turning of retries is what I speculated is the behaviour of kw2xrf, because immediately sensing again makes no sense.